presto-parser/src/main/java/com/facebook/presto/sql/tree/CreateTableAsSelect.java

```diff
@@ -32,26 +32,28 @@
     private final boolean notExists;
     private final Map<String, Expression> properties;
     private final boolean withData;
+    private final Optional<List<String>> columnAliases;
```

Comment:
This should be a list of Identifier, which carry positional information. It will make it possible to report errors more accurately (i.e., the exact line and offset of the alias that's duplicated, etc).